source/03-AT/02.ptx

```diff
@@ -328,6 +328,28 @@ Find the standard matrix for <m>T</m>.
     \end{array}\right]
   </me>
         </p>
+        <p>
+Since the formula for a linear transformation <m>T</m> and
```

Comment:
Explicitly add Ax notation for applying a matrix to a vector.

Reply:
Why is this the right spot for this?  Now we have some matrix multiplication going on before we define matrix multiplication.

Reply:
Thinking more about this, maybe what I am suggesting is to here only introduce $A\vec{x}$, or $A\left[\begin{array}{c}x_1 \\ x_2 \\ x_3 \\ x_4\end{array}\right]$, with a note that the reason for this choice will be explained in Section 4.1.  Then add an activity at the end of 4.1 where they actually multiply the matrix $A$ by the vector $\left[\begin{array}{c}x_1 \\ x_2 \\ x_3 \\ x_4\end{array}\right]$.

Reply:
Agreed - here we're just using $A\vec x$ as a formal notation, and I love the idea of confirming this lines up with matrix multiplication later. Will add that to this PR.

Reply:
see [9d3e273](https://github.com/TeamBasedInquiryLearning/linear-algebra/pull/302/commits/9d3e2738ec99916c8402b0c47efd24295023ff03)